crossbar/controller/processtypes.py

```diff
@@ -104,34 +107,23 @@ def log(self, childFD, data):
         """
         assert(childFD in self._log_fds)
 
-        if _PY3 and type(data) != str:
+        if type(data) != unicode:
             data = data.decode('utf8')
```

Comment:
I think `if type(data) != six.text_type` or `if not isinstance(data, six.text_type)` for consistency with rest of crossbar